buildSrc/src/main/java/dependencies.kt

```diff
@@ -2,28 +2,28 @@ object Versions {
     // Build tools and SDK
     const val buildTools = "30.0.3"
     const val compileSdk = 30
-    const val gradlePlugin = "7.0.0-alpha14"
+    const val gradlePlugin = "7.0.0-beta01"
     const val kotlin = "1.4.32"
     const val minSdk = 16
     const val targetSdk = 29
 
     // Android libraries
-    const val activity = "1.2.0"
+    const val activity = "1.2.3"
     const val activityCompose = "1.3.0-alpha03"
```

Comment:
Might as well update this one too. `1.3.0-alpha07` is the latest, I think. 

```suggestion
    const val activityCompose = "1.3.0-alpha07"
```